python/test/test_action_yml.py

```diff
@@ -1,17 +1,19 @@
+import pathlib
 import unittest
 
 import yaml
-from yaml import Loader
+
+project_root = pathlib.Path(__file__).parent.parent.parent
 
 
 class TestActionYml(unittest.TestCase):
 
     def test_composite_action(self):
-        with open('../../action.yml', encoding='utf-8') as r:
-            dockerfile_action = yaml.load(r, Loader=Loader)
+        with open(project_root / 'action.yml', encoding='utf-8') as r:
+            dockerfile_action = yaml.safe_load(r)
 
-        with open('../../composite/action.yml', encoding='utf-8') as r:
-            composite_action = yaml.load(r, Loader=Loader)
+        with open(project_root / 'composite/action.yml', encoding='utf-8') as r:
+            composite_action =yaml.safe_load(r)
```

Comment:
there are many more places ...

Reply:
Sorry it's getting late my 🧠  stopped for a while 😅